### PR TITLE
Standardize sample attribute descriptions across app

### PR DESCRIPTION
**Pull request by cdebourcy**

### CODE REVIEW

File: app/assets/src/components/utils/SampleFields.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+export const SAMPLE_FIELDS = [
+  { name: "name", label: "Name" },
+  { name: "project_name", label: "Project" },
+  { name: "sample_location", label: "Location" },
+  {
+    name: "created_at",
+    label: "Upload Date",
+    parser: val => {
+      return <Moment fromNow date={val} />;
```
Comment (jshoe) [inline app/assets/src/components/utils/SampleFields.js]: does <Moment> work even though it's not in this file?

Comment (cdebourcy) [reply app/assets/src/components/utils/SampleFields.js]: thanks for catching, fixed

File: app/assets/src/components/PipelineSampleReads.jsx [hunk 2]
```diff
@@ -66,6 +68,41 @@ class PipelineSampleReads extends React.Component {
...
+    ];
+    this.sampleFieldsColumn2 = [
+      "sample_library",
+      "sample_sequencer",
+      "sample_input_pg",
+      "sample_batch",
+      "sample_organism",
+      "sample_detection"
+    ];
+    this.sampleFieldNames = new Map(
```
[the remark this answers is not in the document]
Comment (cdebourcy) [reply app/assets/src/components/PipelineSampleReads.jsx]: ok, renamed to sampleFieldProperties

[the remark this answers is not in the document]
Comment (tfrcarvalho) [reply app/assets/src/components/PipelineSampleReads.jsx]: I think @jshoe meant a name about what the map maps, like somethingToSomething. in this case `sampleFieldToLabel` or something like that...

[the remark this answers is not in the document]
Comment (jshoe) [reply app/assets/src/components/PipelineSampleReads.jsx]: I'm OK with sampleFieldProperties but I do prefer names like sampleFieldNamesToProperties for mappings

**Pull request merged**